src/app/components/BudgetControl/index.tsx

```diff
@@ -27,12 +29,17 @@ function BudgetControl({
 
   return (
     <div className="mb-4">
-      <div className="flex items-center">
+      <div
+        className={`flex items-center ${
+          disabled && "blur-[2px] pointer-events-none select-none"
```

Comment:
Please remove this blur. 

@stackingsaunter Can you please suggest colors for  disabled form inputs?

Reply:
Okay will do it